### DISCUSSION TITLE
Laracord - Create Discord bots with Laravel

### DISCUSSION THREAD

**Discussion by Log1x** [HassanZahirnia/laravel-package-ocean#193] [Package Suggestions]
https://github.com/laracord/laracord

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package! Have a look and let me know if the category and other information is correct.

**Reply by Log1x** [reply]
It looks great! Beautiful site by the way, awesome work. Thank you.

**Reply by HassanZahirnia** [reply]
No worries 😇
